src/org/starexec/command/Connection.java

```diff
@@ -326,8 +326,8 @@ public void logAddTextBody(MultipartEntityBuilder entity, String name, String te
 	// Logs and executes a GET/POST request.
 	private HttpResponse executeGetOrPost(HttpRequestBase request) throws IOException {
 					// -- debug -- vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
-//					System.out.println( "---- now ENTERING executeGetOrPost() --------\n" );
-//					C.debugMode = true;
+					System.out.println( "---- now ENTERING executeGetOrPost() --------\n" );
```

Comment:
It looks like you have debug information on in Connection.java.  Should probably disable before we deploy, right?